add-deck.php

```diff
@@ -0,0 +1,107 @@
+<?php
+require_once 'DeckDAO.php';
+require_once 'functions.php';
+require_once 'Colourid.php';
+require_once 'ColouridDAO.php';
+require_once 'Archetype.php';
+require_once 'ArchetypeDAO.php';
+$deckDao = new DeckDao();
+$colouridDAO = new ColouridDAO();
+$archetypeDAO = new ArchetypeDAO();
+$decks = $deckDao->fetchAll();
+$colourids = $colouridDAO->fetchAll();
+$archetypes = $archetypeDAO->fetchAll();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>MtG: Collector App</title>
+
+    <meta name="description" content="A collector app to show my Magic: The Gathering decks">
+    <meta name="author" content="James McGraw-Allen">
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Special+Elite&display=swap" rel="stylesheet">
+
+    <link rel="icon" href="images/old-sprite.png" sizes="250x250">
+    <link rel="shortcut icon" href="images/old-sprite.png">
+    <link rel="apple-touch-icon" href="images/old-sprite.png">
+</head>
+
+<body>
+<header>
+    <a href="index.php">Go back home</a>
+</header>
+
+<h1>
+    Add a new deck:
+</h1>
+
+<div class="add-deck">
+    <form action="index.php" method="post">
```

Comment:
Suggest adding some validation attributes to this form - for example you can use `required` to make fields non optional. There are a few examples here: https://www.w3schools.com/html/html_form_attributes.asp

Reply:
Added required fields for those that can't have a value of null in DB